src/audio.c

```diff
@@ -472,7 +472,7 @@ static void encode_rtp_send(struct audio *a, struct autx *tx,
 
 		uint32_t rtp_ts = tx->ts_ext & 0xffffffff;
 
-		if (len) {
+		if (len && !tx->muted) {
```

Comment:
Not sending packets if muted can lead to problems like NAT Timeouts etc. Can you explain why this is needed?

Reply:
I was wrong, I had to do two pullrequest
in one opensles optimization
in the second do not send empty packets

Reply:
Yes two PRs are needed, but sending empty packets is a feature not a bug. As said many things rely on sending RTP packets and bad things like NAT/TURN timeouts can happen. On a conference call its normal that one person is muted for minutes.

> When it's mute, it's better not to send anything, bad old androids have a side effect from the microphone to the speaker and vice versa.

Can you explain this further. Do you mean audible noise introduced by network traffic? Mute means all source frames are silenced (overridden with zero).

Reply:
My case is an old Chinese tablet and a bad connection to the intercom.
I have a call going through and if not mute, then the tablet has an echo from its microphone to its speaker. Therefore, it was decided not to send packets when they are not talking on the tablet, but only to receive. Packets are only sent when speech is received on the tablet. Since the connection is bad, the traffic is very important.
I also can’t overcome that even with a good video connection, the picture often crumbles and ripples